roles/smartgateway/tasks/metrics.yml

```diff
@@ -0,0 +1,107 @@
+---
+# tasks file for smartgateway
+- name: Write out configmap for smart gateway
+  template:
+    dest: /tmp/configmap.yaml
+    src: "{{ service_type }}-configmap.yaml.j2"
+
+- name: Smart gateway ConfigMap
+  k8s:
+    src: /tmp/configmap.yaml
+
+- name: Start the smart gateway
+  k8s:
+    resource_definition:
+      apiVersion: apps.openshift.io/v1
+      kind: DeploymentConfig
+      metadata:
+        name: '{{ meta.name }}-smartgateway'
+        namespace: '{{ meta.namespace }}'
+      spec:
+        replicas: '{{ size }}'
+        template:
+          metadata:
+            labels:
+              app: prometheus-{{ meta.name }}
```

Comment:
This one looks suspect to me, though I see it was in the original file before this PR. I'd expect it to be something more like:
`app: smartgateway_metrics_consumer`

Why is this getting a label that makes it look like it's running prometheus?

Reply:
Yea, there are some labeling issues that need to get dealt with. What I suspect is happening is it is being used by the Service Monitor to determine what Prometheus needs to scrape for endpoints. I will review this again and determine if that's the case, and if so, will try and update the labels to be less confusing.